servicetalk-http-api/src/main/java/io/servicetalk/http/api/HttpClientBuilder.java

```diff
@@ -181,6 +184,16 @@
     public abstract HttpClientBuilder<U, R, SDE> serviceDiscoverer(
             ServiceDiscoverer<U, R, ? extends SDE> serviceDiscoverer);
 
+    /**
+     * Sets a retry strategy for errors emitted by {@link ServiceDiscoverer#discover(Object) discovery publisher}.
```

Comment:
`for errors ...` => `to retry errors`

`discovery publisher` => `service discoverer`?